src/Markdig/Extensions/MediaLinks/MediaLinkExtension.cs

```diff
@@ -154,10 +154,12 @@ private bool TryRenderIframeFromKnownProviders(Uri uri, bool isSchemaRelative, H
                 htmlAttributes.AddPropertyIfNotExist("height", Options.Height);
 
             if (!string.IsNullOrEmpty(Options.Class) || !string.IsNullOrEmpty(foundProvider.Class))
-                htmlAttributes.AddPropertyIfNotExist("class",
-                    (!string.IsNullOrEmpty(Options.Class) && !string.IsNullOrEmpty(foundProvider.Class))
-                    ? Options.Class + " " + foundProvider.Class
-                    : Options.Class + foundProvider.Class);
+            {
+                if (!string.IsNullOrEmpty(Options.Class))
+                    htmlAttributes.AddClass(Options.Class);
+                if (!string.IsNullOrEmpty(foundProvider.Class))
+                    htmlAttributes.AddClass(foundProvider.Class);
+            }
```

Comment:
Can you remove the surrounding `if` as it looks duplicated

Reply:
Yep, done.